### DISCUSSION TITLE
同時能比對多種語言的範例程式網站！

### DISCUSSION THREAD

**Discussion by Tuoresa** [1101-GS4719-C/Python-Community-GS4719-C#756] [General]
我發現了一個網站只要輸入關鍵字，就能找到同一主題下，非常多不同程式語言的範例程式！個人覺得在各程式語言間辨認及學習有一些幫助，分享給大家！

網址：https://rosettacode.org/wiki/Rosetta_Code

**Comment by williamyen0623** [comment]
工程師真的都能想出一些酷酷的東西

**Comment by Pietra1226** [comment]
對於程式新手真的很好用，可以看各種不同的範例很方便

**Comment by danwan1027** [comment]
對想學習程式的人的工具又更多了！

**Comment by bonny123865** [comment]
網站的內容好多，好像一個程式版的維基百科一樣!

**Comment by a45320614** [comment]
對於已經掌握一種程式語言想再學習新語言的人來說非常有幫助，而且也不只有基礎的語法，甚至還有一些複雜的演算法問題可以做參考
